publish/cli.go

```diff
@@ -213,6 +231,40 @@ func FXModuleFromFlags(cmd *cobra.Command, debug bool) fx.Option {
 			nats.MaxReconnects(maxReconnect),
 			nats.ReconnectWait(maxReconnectWait),
 		))
+	case sqsEnabled:
+		region, _ := cmd.Flags().GetString(iam.AWSRegionFlag)
+		sqsEndpointOverride, _ := cmd.Flags().GetString(SubscriberSqsEndpointOverrideFlag)
+
+		loadOptions := []func(*config.LoadOptions) error{
+			config.WithRegion(region),
+		}
+		if sqsEndpointOverride != "" {
+			// if we are overriding the endpoint assume we are in a dev context
+			loadOptions = append(loadOptions, config.WithCredentialsProvider(
+				credentials.NewStaticCredentialsProvider("dummy", "dummy", "dummy"),
+			))
+		}
+		cfg, err := config.LoadDefaultConfig(cmd.Context(), loadOptions...)
+		if err != nil {
+			panic(fmt.Sprintf("unable load aws config %v", err))
+		}
+
+		sqsOpts := []func(*sqsservice.Options){}
+		if sqsEndpointOverride != "" {
+			sqsUrl, err := url.Parse(sqsEndpointOverride)
+			if err != nil {
+				panic(fmt.Sprintf("unable to parse sqs url %q", sqsEndpointOverride))
```

Comment:
The parsing could be done in a Provide too, to avoid the panic and let fx handle the error propertly.